# A map of which joints have a real setpoint at this time_from_start.
duration time_from_start
string[] joint_names